TeamCode/src/main/java/org/firstinspires/ftc/teamcode/OpModes/Auto/Basic/Sampling.java

```diff
@@ -0,0 +1,119 @@
+package org.firstinspires.ftc.teamcode.OpModes.Auto.Basic;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.tfod.Recognition;
+import org.firstinspires.ftc.teamcode.Components.IntakeLift;
+import org.firstinspires.ftc.teamcode.OpModes.Auto.Utils.AutoMode;
+
+import java.util.List;
+
+/**
+ * AUTO MODE: Sampling
+ *
+ * POINT WORTH: 15 pts
+ *
+ * SETUP: Front facing the sample
+ *
+ * SUMMARY: The auto uses TF to detect from the sample,
+ *  the position of the gold mineral and turns towards the gold minerals position.
+ *  The robot will then drive forward.
+ */
+
+@Autonomous(name = "Sampling", group = "Official")
+public class Sampling extends AutoMode {
+
+    //Auto specific
+    private boolean sampleComplete = false;
+    private enum Turn{
+        LEFT(-90), RIGHT(90);
+
+        private final int degrees;
+        Turn(int degrees) {
+            this.degrees = degrees;
+        }
+        public int getDegrees() {
+            return degrees;
+        }
+    }
+    Turn turn;
+
+    @Override
+    public void init() {
+        super.init();
+    }
+
+    @Override
+    public void loop() {
+        switch(states){
+            case START:
+                telemetry.addData("Current State","Start");
+                intakeLift.setAutoPos(IntakeLift.AutoPos.UP);
```

Comment:
Not sure if this will actually cause problems, just pointing out that it will not wait for the intake lift to go up, maybe add a slight delay or something? Might not be an issue though

Reply:
@GarrettBurroughs  It was done intentionally to save time, it should not be an issue since we raise it to prevent the issue Tristan brought up and not to actually use the intake since it is still untested